js/ui/select_box.js

```diff
@@ -383,15 +383,15 @@ var SelectBox = DropDownList.inherit({
     },
 
     _renderInputValue: function() {
-        this._renderInputValueAsync = function() {
```

Comment:
@mrjohn0011 

I reverted the `_dispose` method implementation in [this commit](https://github.com/DevExpress/DevExtreme/pull/736/commits/2827189183bbc000dfc4217e22831b66d8ff18ea).